include/TrustWalletCore/TWOrdMimeType.h

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2017-2023 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "TWBase.h"
+
+TW_EXTERN_C_BEGIN
+
+/// The Bitcoin inscription MIME type.
+TW_EXPORT_ENUM(uint32_t)
+enum TWOrdMimeType {
+    TWOrdMimeTypeApplicationJson = 1,
```

Comment:
Will the Bitcoin network reject a BRC20 transaction if the mime type doesn't match this list?
Probably, we could allow the user to pass a mime type as a string? 🤔

Reply:
Yes, we could add a `Custom(String)` variant

Reply:
Although this makes the FFI interface a little more complex... Maybe just pass a string directly, as you said. Will update.